css/style.css

```diff
@@ -0,0 +1,21 @@
+*{
+    font-family: 'Poppins', sans-serif;
+}
+.carousel-item::before{
+    content: "";
+    display: block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background: gray;
+    opacity: 0.7;   
+}
+.card-price-color{
+    color: #289BDE;
+}
+.carousel-image > img {
```

Comment:
It would be better if you give space before using classes. You can use stylelinter to check! 😎

Reply:
Great job with the kind language use 👍  But let's explain why we are requesting this change 👍

Reply:
How about now? @huseyinbiyik

Reply:
Is it god to go now? @huseyinbiyik  @IngDavidTM